docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -121,6 +119,50 @@ This will crawl and download the data from the specified website. If you import
 If you want to prepare you data with custom code, you can use a Jupyter notebook to iterate and test it. NEED LINK
 
 
+## Add a dataset from a S3 bucket
+
+You can try using an arbitrary PRF and HTML data from an S3 bucket with the Sort Benchmark
+importing script. This script is not optimized for new datasets, so the answer quality may
+vary. However we have found positive results with some experiments.
```

Comment:
Some datasets have given good results when queried.
-- Existing However... seems overly negative to me.

Reply:
Ack.